packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -61,6 +61,113 @@ export class RulesetExecutor {
     this.engineRuleset = this.prepareRuleset();
   }
 
+  /**
+   * Plug ruleset to fact streams and trigger a first evaluation
+   *
+   */
+  private prepareRuleset() {
```

Comment:
Would be cleaner to keep the public (as deprecated) field using the private one for pure backward compatible reason (event if calling this function from outside does not really make sense).